### PR TITLE
feat:add text utilites component

### PR DESCRIPTION
**Pull request by shubhamkumar648**
heading
addT ext Utilities
1.headings
2.small text
3.gray text
4.
heading
center text
hed2
update font weight

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
I checked you code and that is good for me

File: component/textutilities.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        <p class="ls-m mt-2 mb-2">
+          Text Utilitiesimproves site usability when text clarity and
+          readability concepts are applied.
+        </p>
+        <h6>Variant of Text: heading|center text|smalltext|gray text</h6>
+        <div class="text__container mb-2">
+          <h1 class="font-xxl">This is heading 1</h1>
+          <h2 class="font-xl">This is heading 2</h2>
+          <h3 class="font-l">This is headding 3</h3>
+          <h4 class="font-m">This is heading 4</h4>
```
Comment (tanishq20) [inline component/textutilities.html:81]: According to the preview, I think `heading 5` has more `font-weight` than `heading-4`.

**Review by tanishq20 — COMMENTED** [review]
Everything looks good. Just a small clarification is needed.

File: component/textutilities.html (hunk 1 above)
Comment (shubhamkumar648) [reply component/textutilities.html:81]: thanks for correction

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**